package-lock.json

```diff
@@ -14162,24 +14162,29 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "bundled": true,
+          "resolved": false,
```

Comment:
you should not push package-lock.json file unless you made changes on package.json